src/ILVerify/tests/ILVerify.Tests.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0-preview-20170427-09" />
+    <PackageReference Include="xunit" Version="2.2.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\ILVerify.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
Yes, I think that happens because of the `Microsoft.NET.Test.Sdk` package. I removed it and VS did not add the tag back. 

> Is that needed? 

I would say yes: without that the Test Explorer in VS does not find the tests and the `dotnet test` command does not work either. Currently this project feels like a natural test project: Test explorer discovers it, `dotnet test` also works and I guess it is easy to integrate it as a test project with other tools. If there is no specific reason to drop `Microsoft.NET.Test.Sdk` other than this `Service` tag I would prefer to keep it.